a10_octavia/db/models.py

```diff
@@ -49,6 +49,7 @@ class VThunder(base_models.BASE):
     updated_at = sa.Column(u'updated_at', sa.DateTime(), nullable=True)
     partition_name = sa.Column(sa.String(14), nullable=True)
     hierarchical_multitenancy = sa.Column(sa.Boolean(), default=False, nullable=True)
+    interface_vlan_map = sa.Column(sa.String(256), nullable=True)
```

Comment:
(Just a thought, not a PR review comment) : 
I do have my doubts about how much it can save, certainly not many map objects. Its fine for now. Ideally we can create a table having foreign references from `vthunders`.